source/daplink/drag-n-drop/flash_decoder.c

```diff
@@ -143,7 +143,15 @@ error_t flash_decoder_get_flash(flash_decoder_type_t type, uint32_t addr, bool a
             }
         } else if (FLASH_DECODER_TYPE_TARGET == type) {
             if (g_board_info.target_cfg) {
+#ifdef TRUSTZONE_M
+                if(addr >= g_board_info.target_cfg->flash_start_alias) {
+                    flash_start_local = g_board_info.target_cfg->flash_start_alias;
+                } else {
+                    flash_start_local = g_board_info.target_cfg->flash_start;
+                }
+#else
                 flash_start_local = g_board_info.target_cfg->flash_start;
+#endif
```

Comment:
TODO:

Add patch for multi regions instead of this logic.

Reply:
Please remove this change and rely on existing multiple flash region support.

Reply:
Will remove this change